### PR TITLE
[wip] Fix loss reduction

### PR DESCRIPTION
**Pull request by justinvyu**

### CODE REVIEW

File: skyrl-train/skyrl_train/workers/worker.py [hunk 1]
```diff
@@ -677,6 +678,10 @@ def forward_backward(self, data: TrainingInputBatch) -> Dict[str, float]:
         for micro_batch in BatchIterator(data, micro_batch_size, drop_last=False):
             metrics = self._forward_backward_micro(micro_batch)
             self._micro_batches_accumulated += 1
+
+            # Track total tokens accumulated to compute average loss per token during optim_step
+            self._total_tokens_accumulated += micro_batch.attention_mask.sum().item()
```
Comment (justinvyu) [inline skyrl-train/skyrl_train/workers/worker.py:683]: this should actually be loss_mask

**Comment by justinvyu** [comment]
close in favor of https://github.com/NovaSky-AI/SkyRL/pull/925
